src/components/Categories/MainCategoryTile.vue

```diff
@@ -128,29 +115,34 @@ export default {
 			}
 			return true;
 		},
-	}
+		altText() {
+			return `Kiva loans to ${this.categoryName}`;
+		},
+		chooseImage() {
```

Comment:
When I tested locally, these images are sometimes blurry in the larger cards. Do we have higher resolution images available?
![image](https://user-images.githubusercontent.com/16867161/176321825-8e780b9a-f772-4353-840e-504eb9f40718.png)

Reply:
@eddieferrer What do you think is happening?

Reply:
In case it helps with diagnosis, I tested with the code from this branch on a 1080p monitor.

Reply:
Good catch @dyersituations 

This is happening because on non-retina monitors, the non-retina image is fetched, and in the legacy admin these images are 320x180, so they are getting scaled up to 520x301 

@galen94 I think for now, lets always always get the retina image for the large cards